docs/quickstart.md

```diff
@@ -8,6 +8,24 @@ To quickly get started using ModelMesh Serving, here is a brief guide.
 - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) (v3.2.0+)
 - At least 4 vCPU and 8 GB memory. For more details, please see [here](install/README.md#deployed-components).
 
+## Recommendation
+
+- [cert-manager](https://github.com/cert-manager/cert-manager)
+
+  ```shell
+  CERT_MANAGER_VERSION="v1.11.0"
+
+  echo "Installing cert manager ..."
+  kubectl create namespace cert-manager
+  sleep 2
+  kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml
+
+  echo "Waiting for cert manager started ..."
+  kubectl wait --for=condition=ready pod -l 'app in (cert-manager,webhook)' --timeout=180s -n cert-manager
+  ```
+
+  _(Note)_ Please refer to [install-script.md](install/install-script.md).
+
```

Comment:
No quick-start does not include `--enable-self-signed-ca`.  `--fvt` include it. 
In order to test FVT, ` --enable-self-signed-ca` is required because installing `cert-manager` by a script is not much. However, for quick-start, we can give options to customers: `cert-manager` or `self-signed-ca` .

Reply:
I meant that the flag `--enable-self-signed-ca` is included [in the quick-start installation command here](https://github.com/kserve/modelmesh-serving/blob/cff39ae1a4180769ab6d9f8157c3149753622f9a/docs/quickstart.md?plain=1#L43). I think that the quick-start should be concise/simple (for experimentation) and not include too many details; the same way we include `--namespace-scope-mode` in the installation command. 

Users would then go to the [install doc](https://github.com/kserve/modelmesh-serving/blob/main/docs/install/install-script.md#prerequisites) for more details/customization if needed. So I think we should treat this similar to `developer.md` where you refer users to the installation doc for more details.

Reply:
updated